build.gradle

```diff
@@ -969,3 +972,84 @@ task releaseAndTest {
 
 // Don't create an empty checker-framework-VERSION.jar
 jar.onlyIf {false}
+
+/**
+ * Deploys the given jar file to the local Maven repository.
+ * @param jar fully qualified file name of a jar file
+ * @param pom pom file describing the artifact
```

Comment:
Is pom a filename, as opposed to the file contents?
(Reading on, I see that it is.)
Same comment a few other places.